package.json

```diff
@@ -190,6 +190,7 @@
     "typedoc": "^0.25.1",
     "typescript": "^5.2.2"
   },
+  "packageManager": "pnpm@10.20.0",
```

Comment:
This works similar to `.tool-versions`

- All collaborators (and CI) use the same version
- Updates are transparent
- Calling `yarn` errors earlier:

```
This project is configured to use pnpm because /path/to/crossnote/package.json has a "packageManager" field`